uat/proto/mqtt_client_control.proto

```diff
@@ -37,6 +37,7 @@ message Mqtt5Message {
     repeated Mqtt5Properties properties = 5;
     optional string contentType = 6;
     optional bool payloadFormatIndicator = 7;
+    optional int32 messageExpireInterval = 8;
```

Comment:
The name should be renamed as messageExpiryInterval

Reply:
Fixed in a81cddb2e537c4cd1aec9699c879c5e855b7cfff